sysl2/sysl/sysl.go

```diff
@@ -106,3 +114,16 @@ func main() {
 		os.Exit(rc)
 	}
 }
+
+//nolint:unparam
+func showBinaryInfo(_ *kingpin.ParseContext) error {
+	fmt.Printf("Version    : %s\n", Version)
+	fmt.Printf("Commit ID  : %s\n", GitCommit)
+	fmt.Printf("Build date : %s\n", BuildDate)
+	fmt.Printf("GoOS       : %s\n", runtime.GOOS)
+	fmt.Printf("GoArch     : %s\n", runtime.GOARCH)
+	fmt.Printf("Golang     : %s\n", GoVersion)
+	fmt.Printf("BuildOS    : %s\n", BuildOS)
+	os.Exit(0)
```

Comment:
This shouldn't be needed. I can see that it's being used to hack the command-line parser. If you really want to do that, supply a lambda that calls showBinaryInfo and then exits. But this really should be implemented as a subcommand, not an option hack.

Reply:
Agree. Changed implementation to use a command instead of an option.
`> sysl info`